compose/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<network-security-config>
```

Comment:
Ohh, do we need to update the Manifest as well?

Reply:
Ah let me remove this file. Good catch!